frontend/UserList.vue

```diff
@@ -17,6 +18,7 @@
           <td>{{user.email}}</td>
           <td>{{user.admin}}</td>
           <td>{{user.disabled}}</td>
+          <td>{{user.roles && user.roles.length ? user.roles.join(', ') : ''}}</td>
```

Comment:
Could also be `(user.roles || []).join(', ')`?

Reply:
👍  I tend to default to Ternary conditionals.